sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -744,6 +744,27 @@ public EventRoute getEventRoute(String eventRouteId) {
         return this.digitalTwinsAsyncClient.getEventRouteWithResponse(eventRouteId, context).block();
     }
 
+    /**
+     * Delete an event route.
+     * @param eventRouteId The Id of the event route to delete.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void deleteEventRoute(String eventRouteId)
+    {
+        deleteEventRouteWithResponse(eventRouteId);
+    }
+
+    /**
+     * Delete an event route.
+     * @param eventRouteId The Id of the event route to delete.
+     * @return A {@link Response} containing no parsed value.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> deleteEventRouteWithResponse(String eventRouteId)
```

Comment:
This should take Context as a parameter no?

Reply:
Yes it should, good catch